docs/development/maintainers.md

```diff
@@ -193,8 +193,27 @@ note to update them independently.
 
 ## Updating pyodide-build
 
-to change the version of pyodide-build, update the PYODIDE_BUILD_REPO and PYODIDE_BUILD_COMMIT
-variables in `Makefile.envs`.
+to change the version of pyodide-build, change the commit of the pyodide-build submodule.
```

Comment:
```suggestion
to change the version of pyodide-build, change the commit hash of the pyodide-build submodule.
```

(nit)

Reply:
I think the original text sounds better to me. `commit hash` feels like too low of an abstraction level for this sentence.

Reply:
Ah, that's alright. I suggested "commit hash" because @ryanking13 has used it below in the same file: "to test with the fork of pyodide-build, change the `.gitmodules` file to point to your fork and update the commit hash". But in the context of a submodule, a "commit" might sound better ("updating a submodule to the latest commit" and so on).